### PR TITLE
[LOPS-1952] new relic agent update release note

### PR DESCRIPTION
**Pull request by scottbuscemi**
Closes #

## Summary

Release Note for New Relic agent update

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [x] After PHP 8.3 compatibility has been rolled out for the New Relic agent (Scott will mark complete here)

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)

### CODE REVIEW

File: source/releasenotes/2024-04-24-new-relic-agent-update-released.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+The New Relic PHP agent has been updated to the latest version (v10.19.0.9). Compared with the previously available version on the platform (v10.10.0.1), some of the changes and fixes include:
+
+* Drupal 9.4+ now reports hooks and modules (see below for more details)
+* Resolved erroneous deprecation warnings for PHP 8.1+
+* Support for PHP 8.3
+
+See the [complete New Relic PHP agent changelog](https://docs.newrelic.com/docs/release-notes/agent-release-notes/php-release-notes/) for more details.
+
+[Learn More about New Relic](https://docs.pantheon.io/guides/new-relic), real-time performance monitoring for your Pantheon web applications.
```
Comment (stevector) [inline source/releasenotes/2024-04-24-new-relic-agent-update-released.md:15]:
```suggestion
[Learn More about New Relic](/guides/new-relic), real-time performance monitoring for your Pantheon web applications. 
```

File: source/releasenotes/2024-04-24-new-relic-agent-update-released.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* Resolved erroneous deprecation warnings for PHP 8.1+
+* Support for PHP 8.3
+
+See the [complete New Relic PHP agent changelog](https://docs.newrelic.com/docs/release-notes/agent-release-notes/php-release-notes/) for more details.
+
+[Learn More about New Relic](https://docs.pantheon.io/guides/new-relic), real-time performance monitoring for your Pantheon web applications. 
+
+## Action Required: Default behavior of New Relic Drupal Hooks metrics has changed
+
+Pantheon no longer supports Drupal Hooks metrics in New Relic by default. However, the functionality is still available. If this is crucial for troubleshooting your sites, please add these lines to your site’s `pantheon.yml` file to enable this reporting:
```
Comment (stevector) [inline source/releasenotes/2024-04-24-new-relic-agent-update-released.md:19]:
```suggestion
Pantheon no longer logs Drupal Hooks metrics in New Relic by default. However, the functionality is still available. If this is crucial for troubleshooting your sites, please add these lines to your site’s `pantheon.yml` file to enable this reporting:
```

Comment (stevector) [reply source/releasenotes/2024-04-24-new-relic-agent-update-released.md:19]: @scottbuscemi is it really just Drupal _hooks_ or all Drupal data? To my recollection, New Relic logs data about at least hooks, modules, and Views.

Comment (stevector) [reply source/releasenotes/2024-04-24-new-relic-agent-update-released.md:19]: Yeah https://docs.newrelic.com/docs/apm/agents/php-agent/frameworks-libraries/drupal-specific-functionality/

**Review by stevector — DISMISSED** [review]
@scottbuscemi can you accept these minor tweaks and answer the Drupal question?

**Comment by scottbuscemi** [comment]
@stevector Updated the note :)

File: source/releasenotes/2024-05-07-new-relic-agent-update-released.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: New Relic PHP agent v10.19.0.9 now available with PHP 8.3 compatibility and other fixes
+published_date: "2024-05-07"
+categories: [tools-apis]
```
Comment (pwtyler) [inline source/releasenotes/2024-05-07-new-relic-agent-update-released.md:4]:
```suggestion
categories: [tools-apis, action-required]
```
If we're going to use an action required heading, should tag it as such.

**Review by pwtyler — APPROVED** [review]

File: source/releasenotes/2024-05-07-new-relic-agent-update-released.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+title: New Relic PHP agent v10.19.0.9 now available with PHP 8.3 compatibility and other fixes
+published_date: "2024-05-07"
```
Comment (jazzsequence) [inline source/releasenotes/2024-05-07-new-relic-agent-update-released.md:3]:
```suggestion
published_date: "2024-05-06"
```

**Pull request merged**